src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1508,6 +1514,23 @@ namespace ReportSizingManager {
 						}
 					}
 					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirInletTempSizing ) {
+					if ( DesicDehum( DataDesicDehumNum ).RegenInletIsOutsideAirNode ) {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatOutTemp;
+					} else {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatRetTemp;
+					}
+					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirOutletTempSizing ) {
+					AutosizeDes = DesicDehum( DataDesicDehumNum ).RegenSetPointTemp;
+					bCheckForZero = false;
+				} else if ( SizingType == DesiccantRegCoilDesAirInletHumRatSizing ) {
+					if ( DesicDehum( DataDesicDehumNum ).RegenInletIsOutsideAirNode ) {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatOutHumRat;
+					} else {
+						AutosizeDes = FinalSysSizing( CurSysNum ).HeatRetHumRat;
+					}
+					bCheckForZero = false;
```

Comment:
Replaced the desiccant specific sizing methods to general as suggested as follows:
HeatingCoilDesAirInletTempSizing;
HeatingCoilDesAirOutletTempSizing
HeatingCoilDesAirInletHumRatSizing;